src/main/java/com/microsoft/azure/batch/protocol/models/ExitOptions.java

```diff
@@ -18,23 +18,36 @@
      * An action to take on the job containing the task, if the task completes
      * with the given exit condition and the job's onTaskFailed property is
      * 'performExitOptionsJobAction'.
-     * The default is none for exit code 0 and terminate for all other exit
-     * conditions. If the job's onTaskFailed property is noAction, then specify
-     * this property returns an error. The add task request fails with an
-     * invalid property value error;; if you are calling the REST API directly,
-     * the HTTP status code is 400 (Bad Request). Possible values include:
-     * 'none', 'disable', 'terminate'.
+     * Values are:
+     *
+     * none - Take no action.
+     * disable - Disable the job. This is equivalent to calling the disable job
+     * API, with a disableTasks value of requeue.
+     * terminate - Terminate the job. The terminateReason in the job's
+     * executionInfo is set to "TaskFailed". The default is none for exit code
+     * 0 and terminate for all other exit conditions.
+     *
+     * If the job's onTaskFailed property is noAction, then specifying this
+     * property returns an error and the add task request fails with an invalid
+     * property value error; if you are calling the REST API directly, the HTTP
+     * status code is 400 (Bad Request). Possible values include: 'none',
+     * 'disable', 'terminate'.
      */
     @JsonProperty(value = "jobAction")
     private JobAction jobAction;
 
     /**
      * An action that the Batch service performs on tasks that depend on this
      * task.
+     * Values are:
+     *
+     * satisfy - Satisfy the task's dependencies.
+     * block - Block the task's dependencies.
+     *
      * The default is 'satisfy' for exit code 0, and 'block' for all other exit
      * conditions. If the job's usesTaskDependencies property is set to false,
-     * then specifying the dependencyAction property returns an error. The add
-     * task request fails with an invalid property value error; if you are
+     * then specifying the dependencyAction property returns an erro and the
```

Comment:
Looks like we have a typo in the swagger spec: "erro"

Reply:
Should let Matt known it. Swagger tool should have spelling check.